### PR TITLE
Give warning for descriptions >78 instead of killing process

### PR DESCRIPTION
**Pull request by onurctirtir**

### CODE REVIEW

File: automated_packaging/common_functions.pm [hunk 1]
```diff
@@ -151,12 +151,13 @@ sub create_release_changelog {
                 if ($line =~ /^DESCRIPTION: */) {
                     $description_part = substr($line, length($&), -1);
                     if (length($description_part) > 78) {
-                        print("You have to shorten PR message $description_part of $pr_url");
-                        `git reset --hard`;
-                        die "Can not add description longer than 78 charachters";
+                        print("You have to shorten PR message $description_part of $pr_url\n");
+                        print("Can not add description longer than 78 charachters, please manually add this description\n");
```
Comment (hanefi) [inline automated_packaging/common_functions.pm:155]: I suggest you add the warnings to the CHANGELOG as well.  It should be clear which entry is longer than the allowed 78 chars.

Comment (onurctirtir) [reply automated_packaging/common_functions.pm:155]: hmm, you suggest me to add those ones to changelog but with a warning, makes sense

Comment (onurctirtir) [reply automated_packaging/common_functions.pm:155]: then we manually remove those warnings and shorten those descriptions

File: automated_packaging/common_functions.pm [hunk 2]
```diff
@@ -150,11 +150,12 @@ sub create_release_changelog {
             foreach $line (@log_output) {
                 if ($line =~ /^DESCRIPTION: */) {
                     $description_part = substr($line, length($&), -1);
+                    
                     if (length($description_part) > 78) {
-                        print("You have to shorten PR message $description_part of $pr_url");
-                        `git reset --hard`;
-                        die "Can not add description longer than 78 charachters";
+                        print("You have to shorten PR message $description_part of $pr_url\n");
+                        print("Description should not be longer than 78 charachters, please manually shorten this description\n");
```
Comment (hanefi) [inline automated_packaging/common_functions.pm:156]:
```suggestion
                        print("Description should not be longer than 78 charachters, please manually shorten this description\n");
                        push(@comment_lines, "* " . "PLEASE SHORTEN THE NEXT LINE MANUALLY, IT SHOULD BE NO LONGER THAN 78 CHARS\n");
```

File: automated_packaging/common_functions.pm (hunk 1 above)
Comment (hanefi) [reply automated_packaging/common_functions.pm]: Please see https://github.com/citusdata/tools/pull/74#discussion_r422870340

File: automated_packaging/common_functions.pm (hunk 2 above)
Comment (onurctirtir) [reply automated_packaging/common_functions.pm:156]: Oh now I understood what you purposed -adding those warnings to changelog as well-, makes sense

**Pull request merged**